app/views/notes/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with model: [ current_user, @note ] do |form| %>
```

Comment:
we don't spaces here, it should be  [current_user, @note].
We add space for curly braces { current_user, @note } but not for []

Reply:
done